### PR TITLE
Create symbol_inception-v4.py

### PR DESCRIPTION
**Pull request by wac81**
Create symbol_inception-v4.py

### CODE REVIEW

File: example/image-classification/symbols/symbol_inception-v4.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
+# -*- coding:utf-8 -*-
```
Comment (mli) [inline example/image-classification/symbols/symbol_inception-v4.py]: rename from `symbol_inception-v4.py` to `inception-v4.py`

File: example/image-classification/symbols/symbol_inception-v4.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+Inception v4 , suittable for image with around 299 x 299
+
+Reference:
+    Inception-v4, Inception-ResNet and the Impact of Residual Connections on Learning
+    Christian Szegedy, Sergey Ioffe, Vincent Vanhoucke
+    arXiv.1602.07261
+
+
+'''
+import find_mxnet
```
Comment (mli) [inline example/image-classification/symbols/symbol_inception-v4.py]: not necessary

File: example/image-classification/symbols/symbol_inception-v4.py [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    c4 = Conv(c4, 512, kernel=(3, 1), pad=(1, 0), name='%s_conv8_3*1' %name)
+    c4_1 = Conv(c4, 256, kernel=(3, 1), pad=(1, 0), name='%s_conv9_1*3' %name)
+    c4_2 = Conv(c4, 256, kernel=(1, 3), pad=(0, 1), name='%s_conv10_3*1' %name)
+
+    concat = mx.symbol.Concat(*[c1, c2, c3_1, c3_2, c4_1, c4_2], name='%s_concat' %name)
+
+    return concat
+
+
+def get_symbol(num_classes=1000):
```
Comment (mli) [inline example/image-classification/symbols/symbol_inception-v4.py]: add `**kwargs` after `num_class=1000`

**Comment by sxjscience** [comment]
What's the difference between https://github.com/dmlc/mxnet/blob/master/example/image-classification/symbols/inception-resnet-v2.py and this network?

File: example/image-classification/symbols/symbol_inception-v4.py (hunk 1 above)
Comment (wac81) [inline example/image-classification/symbols/symbol_inception-v4.py]: yeah, we trained using this network. but not reproduce the numbers, not for val_acc.
this network is copy from paper entirely.

**Comment by wac81** [comment]
@sxjscience v4 with out resnet structure, just deeper network than v3 and v2

File: example/image-classification/symbols/symbol_inception-v4.py (hunk 3 above)
Comment (wac81) [inline example/image-classification/symbols/symbol_inception-v4.py]: thank you

**Comment by piiswrong** [comment]
Can one of the admins verify this patch?

**Comment by mli** [comment]
can you resolve the code review comments?

**Comment by wac81** [comment]
i can't resolve cause i can't see any info about pr-merge 
 
  
image

**Comment by wac81** [comment]
@mli no problem right now

**Comment by wac81** [comment]
@mli when merge to master branch?

**Comment by mli** [comment]
I'm going to close the PR since there is no activity for more than 2 months. But please feel free to re-open it and update with the master.

**Pull request closed without merging**